### ISSUE TITLE
Feature request: Gracefully close miner

### ISSUE THREAD

**Issue by barisahmet** [billiford/procswap#13]
I am using NiceHash QuickMiner, which applies overclocking settings when I run, and resets them when I close it. But when I use procswap, it doesn't reset my card settings for gaming. So, I guess procswap just kills the process right?

Can we have an option for gracefully close miner? It would be useful for users like me.

Thanks!

**Comment by billiford** [comment]
Yep, procswap is just killing the process.

Do you use Afterburner for your overclocking settings or is this something that NiceHash QuickMiner does for you? If you use Afterburner all you need is to run a script when a game opens, which is possible using the `--priority-script` flag. Its setup is super simple and you can read more about it in the README [here](https://github.com/billiford/procswap#switching-afterburner-profiles-automatically).

If that doesn't solve it for you can you send me the following:
1) your procswap command
2) contents of batch files that you run (if any, and feel free to redact mining address info if you want)
3) your NiceHash configuration (or just describe it to me)

I'd be happy to get to this when I have the time, any questions please feel free to ask!

**Comment by barisahmet** [comment]
No, I don't use Afterburner, NiceHash QuickMiner is applying overclocking settings for me upon launch, then resets them when I gracefully exit app without Afterburner.

1. My procswap command is as simple as: 
`cd "C:\Program Files\NiceHash\NiceHashQuickMiner\"`
`"%USERPROFILE%\Dropbox\Applications\ProcSwap\procswap.exe" --priority "C:\Program Files (x86)\Steam\steamapps\common"  --swap "C:\Program Files\NiceHash\NiceHashQuickMiner\NiceHashQuickMiner.exe"`
(had to run quickminer from it's directory)

2. I don't have any batch files to run.
3. I simply use [NiceHash QuickMiner](https://github.com/nicehash/NiceHashQuickMiner), and set custom overclocking profile (you can right click quickminer icon bottom right corner and choose "OCTune"). After that its simple as run and quit from app, it applies all settings by itself.

Note that quickminer is running as Administrator by default to apply some settings, idk if it's possible to do what I want. But it would be good if you can implement it, too many people started using quickminer and it will help a lot.

Thanks for the suggestion tho, I think I will use it with `--priority-script` for now.

**Comment by billiford** [comment]
I think you can get around this with a priority script, but I won't close this issue until I put some time into figuring out if I can gracefully close an exe file.

Thanks for all the info, that's plenty go on.
